<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>end</class>
 <widget class="QWidget" name="end">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>276</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Candara</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Rejouer</string>
  </property>
  <widget class="QPushButton" name="btRestart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">:focus {background-color:#CECECE;border-radius:10px;outline:none;font :bold;}
:!focus{background-color: rgb(255, 255, 255);border-radius:10px}</string>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">:focus {background-color:#CECECE;border-radius:10px;outline:none;font :bold;}
:!focus{background-color: rgb(255, 255, 255);border-radius:10px}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLabel" name="lbScore">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Verdana&quot;;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lbRNbScore">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Verdana&quot;;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="lbRNomJoueur">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font:12pt &quot;Verdana&quot;;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font:12pt &quot;Verdana&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Dommage, vous venez de perdre!
 Allez encore un effort,  vous pouvez faire mieux!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
